README.md

```diff
@@ -67,7 +67,7 @@ You can explicitly define the required jdk version (in system.properties)
 
 ```
 heroku {
-	jdkVersion        = 12
+  jdkVersion = 11
```

Comment:
Using an unsupported feature version as an example did not seem to be a good idea. `11` will be supported for the foreseeable future.